linalg/strumpack.hpp

```diff
@@ -72,138 +80,192 @@ template <typename STRUMPACKSolverType>
-    *    COMBBLAS:                      Use AWPM from CombBLAS (only with
+    *  - COMBBLAS:                      Use AWPM from CombBLAS (only with
     *                                   version >= 3)
     */
    void SetMatching(strumpack::MatchingJob job);
 
-   /**
+   /** @brief Select compression for sparse data types
+    *
     * Enable support for rank-structured data formats, which can be used
     * for compression within the sparse solver.
     *
     * Supported compression types are:
-    *    NONE:           No compression, purely direct solver (default)
-    *    HSS:            HSS compression of frontal matrices
-    *    BLR:            Block low-rank compression of fronts
-    *    HODLR:          Hierarchically Off-diagonal Low-Rank
+    *  - NONE:           No compression, purely direct solver (default)
+    *  - HSS:            HSS compression of frontal matrices
+    *  - BLR:            Block low-rank compression of fronts
+    *  - HODLR:          Hierarchically Off-diagonal Low-Rank
     *                    compression of frontal matrices
-    *    BLR_HODLR:      Block low-rank compression of medium
+    *  - BLR_HODLR:      Block low-rank compression of medium
     *                    fronts and Hierarchically Off-diagonal
     *                    Low-Rank compression of large fronts
-    *    ZFP_BLR_HODLR:  ZFP compression for small fronts,
+    *  - ZFP_BLR_HODLR:  ZFP compression for small fronts,
     *                    Block low-rank compression of medium
     *                    fronts and Hierarchically Off-diagonal
     *                    Low-Rank compression of large fronts
-    *    LOSSLESS:       Lossless compression
-    *    LOSSY:          Lossy compression
+    *  - LOSSLESS:       Lossless compression
+    *  - LOSSY:          Lossy compression
     *
     * For versions of STRUMPACK < 5, we support only NONE, HSS, and BLR.
     * BLR_HODLR and ZPR_BLR_HODLR are supported in STRUMPACK >= 6.
     */
    void SetCompression(strumpack::CompressionType type);
+
+   /** @brief Set the relative tolerance for low rank compression methods
+    *
+    * This currently affects BLR, HSS, and HODLR.  Use
+    * STRUMPACKSolverBase::SetCompression to set the proper compression type.
+    */
    void SetCompressionRelTol(double rtol);
+
+   /** @brief Set the absolute tolerance for low rank compression methods
+    *
+    * This currently affects BLR, HSS, and HODLR.  Use
+    * STRUMPACKSolverBase::SetCompression to set the proper compression type.
+    */
    void SetCompressionAbsTol(double atol);
+
 #if STRUMPACK_VERSION_MAJOR >= 5
+   /** @brief Set the precision for the lossy compression option
+    *
+    * Use STRUMPACKSolverBase::SetCompression to set the proper compression type.
+    */
    void SetCompressionLossyPrecision(int precision);
+
+   /** @brief Set the number of butterflylevels for the HODLR compression option
```

Comment:
minor:  butterflylevels -> "butterfly levels" (e.g. two words in https://petsc.org/release/manualpages/Mat/MatSTRUMPACKSetCompButterflyLevels/)